src/backend/core/tests/team_accesses/test_api_team_accesses_list.py

```diff
@@ -53,20 +53,27 @@ def test_api_team_accesses_list_authenticated_unrelated():
 def test_api_team_accesses_list_authenticated_related():
     """
     Authenticated users should be able to list team accesses for a team
-    to which they are related, whatever their role in the team.
+    to which they are related, with a given role.
     """
     identity = factories.IdentityFactory(is_main=True)
     user = identity.user
 
     team = factories.TeamFactory()
-    user_access = models.TeamAccess.objects.create(team=team, user=user)  # random role
 
-    # other team members should appear
-    other_member = factories.UserFactory()
-    other_member_identity = factories.IdentityFactory(is_main=True, user=other_member)
-    access1 = factories.TeamAccessFactory.create(team=team, user=other_member)
+    member1 = factories.IdentityFactory(is_main=True)
```

Comment:
user the word "user" or use the name of his role?
```suggestion
    owner = factories.IdentityFactory(is_main=True).user
```